src/main/java/racingcar/verifier/InputVerifier.java

```diff
@@ -0,0 +1,61 @@
+package racingcar.verifier;
+
+import java.util.List;
+import racingcar.system.ExceptionMessage;
+import racingcar.system.SystemConstant;
+
+public class InputVerifier {
+    public static void verifyName(String names) throws IllegalArgumentException {
+        InputVerifier.checkEmpty(names);
+        List<String> nameList = List.of(names.split(","));
+        nameList.forEach(InputVerifier::checkLength);
+        nameList.forEach(InputVerifier::checkNull);
+        checkDistinct(nameList);
+    }
+
+    public static void verifyTry(String tryRemain) throws IllegalArgumentException {
+        checkEmpty(tryRemain);
+        checkNumeric(tryRemain);
+        checkNonPositive(tryRemain);
+    }
+
+    private static void checkLength(String name) {
+        if (name.length() > SystemConstant.MAX_NAME_LENGTH.getValue()) {
+            throw new IllegalArgumentException(ExceptionMessage.OUT_OF_RANGE.getMessage());
+        }
+    }
+
+    private static void checkNull(String name) {
+        if (name.isEmpty()) {
+            throw new IllegalArgumentException(ExceptionMessage.IS_NULL.getMessage());
+        }
+    }
+
+    private static void checkDistinct(List<String> nameList) {
+        if (nameList.stream()
+                .distinct()
+                .count() != nameList.size()) {
```

Comment:
이건 제 생각이고 틀릴 수도 있는데 저는 이렇게 조건문 안에 들어가는 식이 복잡해지면 
하나의 메서드로 따로 빼는 것도 좋아보입니다.
```suggestion
        if (isDuplicate()) {
```

Reply:
확실히 가독성이 좋아보이네요 좋은 지적 감사합니다! ☺️